### DISCUSSION TITLE
I'm looking for the logfile for events like player join, player death, etc

### DISCUSSION THREAD

**Discussion by scotttball** [mbround18/valheim-docker#416] [Help!]
I'm looking for the logfile for the stats on the server and I can't find it. Is there a flag that I can pass somewhere to output the server status (player join, player death, etc)?

Thanks for the help!

**Comment by mbround18** [comment]
> 
> 
> I'm looking for the logfile for the stats on the server and I can't find it. Is there a flag that I can pass somewhere to output the server status (player join, player death, etc)?
> 
> Thanks for the help!

Hey I am so sorry this took so long to respond to, I was out of town. Log files should be under `/home/steam/valheim/logs`

**Reply by Aetrius** [reply]
I'm attempting to find the player stats but it appears that these are not visible in those logs. I have a discord bot trying to run on the host using lightsail but when it connects to the mapped logs for server logs it doesn't appear that player deaths are being logged.

**Reply by mbround18** [reply]
I do not think they are in general, that would be a feature for the Valheim game as I sadly cannot tap into the servers data streams :(

**Reply by mbround18** [reply]
@Aetrius you might be able to find a mod to supply that kind of data?
